static/src/scss/_home.scss

```diff
@@ -1,6 +1,8 @@
 body.home {
   section.home-hero {
-    background-color: $gray-800;
+    background-image: url("/site_media/static/images/hedera-ivy.png");
```

Comment:
This file hasn't been added to source control.  Can you please add it @arthurian?

Reply:
oh looks like it's a `jpg`

Reply:
Good catch @paltman. I think this should be fixed now.